src/index.html

```diff
@@ -5,10 +5,37 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+          rel="stylesheet">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+          href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="https://www.google.com.ua/?hl=ru" class="logo">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+      <nav class="nav">
+        <ul class="navbar">
+          <li class="navlist"><a href="https://www.google.com.ua/?hl=ru" class="navlink apple is-active">APPLE</a></li>
+          <li class="navlist"><a href="https://www.google.com.ua/?hl=ru" class="navlink sumsung ">SUMSUNG</a></li>
+          <li class="navlist"><a href="https://www.google.com.ua/?hl=ru" class="navlink smartphone ">SMARTPHONES</a></li>
+          <li class="navlist">
+            <a href="https://www.google.com.ua/?hl=ru"
+               class="navlink lap_com"
+               data-qa="hover">LAPTOPS & COMPUTERS</a>
+          </li>
```

Comment:
1. apply code styling recommendations from a checklist to other items as well
2. use BEM methodology for naming classes
3. no needs to set a link. You can just use `#`
4. use lowercase at html and change this value at css
```suggestion
          <li class="nav__item">
            <a 
              href="#laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
              laptops & computers
            </a>
          </li>
```